packages/wearable_rotary/lib/wearable_rotary.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/services.dart';
+
+const String _ROTARY_CHANNEL_NAME = 'flutter.wearable_rotary.channel';
```

Comment:
```suggestion
const String _channelName = 'flutter.wearable_rotary.channel';
```
https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names